todo_app/templates/index.html

```diff
@@ -11,8 +11,17 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in to_do_items %}
+          <li>{{ item.title }}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+
+  <form action="/submit" method="post">
+    <h2>Add a new item</h2>
+    <label for="title">Title:</label>
+    <input type="text" id="title" name="title">
```

Comment:
Really minor, but adding a `required` field to this input just to take advantage of browser validation would be a nice touch; obviously if we really care we should also add backend validation but as a small touch of UX it's an easy thing